docs/source/app_developers_guide/intro_xo_transaction_family.rst

```diff
@@ -2,191 +2,166 @@
+     The ``xo`` client sends requests to update and query the blockchain to the
+     URL of the REST API (by default, ``http://127.0.0.1:8008``).
 
-Start the Necessary Components
-------------------------------
+     If the REST API's URL is not ``http://127.0.0.1:8008``, you must add the
+     ``--url`` argument to each ``xo`` command in this procedure.
 
-To play *XO*, ensure that the following components are running and connected:
 
-#. At least one validator
-#. An *XO* family transaction processor
-#. The REST API
-
-Create Players
---------------
+Step 1. Create Players
+----------------------
 
 Create keys for two players to play the game:
 
 .. code-block:: console
 
     $ sawtooth keygen jack
-    $ sawtooth keygen jill
-
-
-The command produces output similar to the following for both players:
-
-.. code-block:: console
-
     writing file: /home/ubuntu/.sawtooth/keys/jack.priv
     writing file: /home/ubuntu/.sawtooth/keys/jack.addr
+
+    $ sawtooth keygen jill
     writing file: /home/ubuntu/.sawtooth/keys/jill.priv
     writing file: /home/ubuntu/.sawtooth/keys/jill.addr
 
 
-Create a Game
--------------
+.. note::
+
+   The output may differ slightly from this example.
+
 
-Create a game with the following command:
+Step 2. Create a Game
+---------------------
+
+Create a game named ``game`` with the following command:
 
 .. code-block:: console
 
     $ xo create game --username jack
 
-To see list of the created games, enter the following command:
+To specify a non-default URL for the REST API, add the ``url`` argument
+to this and all following ``xo`` commands.  This example shows the URL for
+the Docker application environment, ``http://rest-api:8008``.
 
 .. code-block:: console
 
-    $ xo list
+   $ xo create game --username jack --url http://rest-api:8008
+
+.. note::
+
+   The ``--username`` argument is required for ``xo create`` and ``xo take``
+   so that a single user (you) can play as two players. By default,
+   ``<username>`` is the player's user name.
 
-The command outputs a list of the games that have been created:
+   An optional ``--key-dir`` argument can be used to specify a non-default
+   location for the user's private key file. By default, this file is at
+   ``<key-dir>/<username>.priv``.
+
+Verify that the game was created by displaying the list of existing games:
 
 .. code-block:: console
 
+    $ xo list
     GAME            PLAYER 1        PLAYER 2        BOARD     STATE
     game                                            --------- P1-NEXT
 
 
-Take a Space as Player One
---------------------------
+Step 3. Take a Space as Player 1
+--------------------------------
 
-Start playing by taking a space as the first player, "jack":
+The first player to issue an ``xo take`` command to a newly created game is
+recorded by username as ``PLAYER 1``. The second player to issue a ``take``
+command is recorded by username as ``PLAYER 2``.
```

Comment:
as above I think the reader may benefit from explaining that these commands are issuing transactions - unless this sort of background is given in a different point in the tutorial?

Reply:
This is another TO DO SOON item.

Reply:
I'll add this info in the introduction to "Playing XO with the xo Client". It's just a simple sentence for now, with more explanation on my to-do list for the next update.